app/Jobs/Notifications/ChannelAnnouncement.php

```diff
@@ -7,17 +7,10 @@
 
 namespace App\Jobs\Notifications;
 
-use App\Models\Notification;
-
 class ChannelAnnouncement extends ChannelMessageBase
 {
     const DELIVERY_MODE_DEFAULTS = ['mail' => true, 'push' => true];
 
-    public static function getBaseKey(Notification $notification): string
```

Comment:
The mail key groups are supposed to match the push notification keys, otherwise the function should be renamed

Reply:
That plan definitely has failed a long time ago.